lib/plug/static.ex

```diff
@@ -117,17 +117,7 @@ defmodule Plug.Static do
   end
 
   def init(opts) do
-    at = Keyword.fetch!(opts, :at)
     from = Keyword.fetch!(opts, :from)
-    gzip? = Keyword.get(opts, :gzip, false)
-    brotli? = Keyword.get(opts, :brotli, false)
-    only = Keyword.get(opts, :only, [])
-    prefix = Keyword.get(opts, :only_matching, [])
-
-    qs_cache = Keyword.get(opts, :cache_control_for_vsn_requests, "public, max-age=31536000")
-    et_cache = Keyword.get(opts, :cache_control_for_etags, "public")
-    et_generation = Keyword.get(opts, :etag_generation, nil)
-    headers = Keyword.get(opts, :headers, %{})
 
     from =
       case from do
```

Comment:
@lexmag But we need that temporal result in the subsequent calls, maybe we should do: `case from = Keyword.fetch!(opts, :from) do`